radicle-cli/src/commands/remote/rm.rs

```diff
@@ -0,0 +1,11 @@
+use crate::git;
+use crate::terminal as term;
+
+pub fn run(repository: &git::Repository, alias: &str) -> anyhow::Result<()> {
```

Comment:
We should be consistent with the naming and since we've gone with `name` elsewhere:
```suggestion
pub fn run(repository: &git::Repository, name: &str) -> anyhow::Result<()> {
```